diff --git a/dist/config.d.ts b/dist/config.d.ts
index cada1fadd79ad5ec6dddce7a34eb46bd995d923c..708d714ca38fcc67a01d7d4ff35de8f1144006ae 100644
--- a/dist/config.d.ts
+++ b/dist/config.d.ts
@@ -7,7 +7,7 @@ export interface RemixMdxConfig {
 export declare type RemixMdxConfigFunction = (filename: string) => Promise<RemixMdxConfig | undefined> | RemixMdxConfig | undefined;
 export declare type ServerBuildTarget = "node-cjs" | "arc" | "netlify" | "vercel" | "cloudflare-pages" | "cloudflare-workers" | "deno";
 export declare type ServerModuleFormat = "esm" | "cjs";
-export declare type ServerPlatform = "node" | "neutral";
+export declare type ServerPlatform = "node" | "neutral" | "browser";
 declare type Dev = {
     port?: number;
     appServerPort?: number;
diff --git a/dist/config.js b/dist/config.js
index c0667a43453e39bc164a8e1f1984b873d9b36cca..920a2a9ca539b15640326b1e68e6d7f7daf5386e 100644
--- a/dist/config.js
+++ b/dist/config.js
@@ -90,7 +90,7 @@ async function readConfig(remixRoot, serverMode = serverModes.ServerMode.Product
     case "cloudflare-workers":
     case "deno":
       serverModuleFormat = "esm";
-      serverPlatform = "neutral";
+      serverPlatform = "browser";
       break;
   }
   let mdx = appConfig.mdx;
