diff --git a/package.json b/package.json
index d69f6cd396e0c0db4fa3bc22243edbb909ba7b79..df2b3a64e6e6e4ea5ecedc169ffaf7c7813201ad 100644
--- a/package.json
+++ b/package.json
@@ -10,8 +10,8 @@
   "type": "module",
   "types": "./types/index.d.ts",
   "module": "./src/index.js",
-  "main": "./dist/pureimage-umd.cjs",
-  "browser": "./src/browser.js",
+  "main": "./dist/index.js",
+  "browser": "./src/index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/joshmarinacci/node-pureimage.git"
diff --git a/src/bitmap.js b/src/bitmap.js
index 31e6278737f1ace567f35fe850ae89c0e42b8b46..5b2263593c897d12c6b169ab000b59fde267ca83 100644
--- a/src/bitmap.js
+++ b/src/bitmap.js
@@ -33,7 +33,7 @@ export class Bitmap {
         /**
          * @type {ArrayBuffer}
          */
-        this.data = Buffer.alloc(w*h*4);
+        this.data = new Uint8Array(w*h*4);
 
         // per the spec, fill it with opaque black
         // https://html.spec.whatwg.org/multipage/canvas.html#output-bitmap
diff --git a/src/index.js b/src/index.js
index a6b46d05815541dde711d39bf7bb81d7eedd704a..01393030e44a49f56861cc5ea426254a10389477 100644
--- a/src/index.js
+++ b/src/index.js
@@ -1,5 +1,5 @@
 import {Bitmap} from './bitmap.js'
-import {PNG} from "pngjs"
+import {PNG} from "pngjs/browser"
 import * as JPEG from "jpeg-js"
 import {getBytesBigEndian} from './uint32.js'
 
